src/App.jsx

```diff
@@ -14,45 +16,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_METOD_NAME = 'name';
+const SORT_METOD_LENGTH = 'length';
+
+function getPreparedGoods(goods, sortMetod, reverse) {
+  const preparedGoods = [...goods];
+
+  if (sortMetod) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortMetod) {
+        case SORT_METOD_NAME:
+          return good1.localeCompare(good2);
+
+        case SORT_METOD_LENGTH:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (reverse) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
 
-    <ul>
-      <li data-cy="Good">Dumplings</li>
-      <li data-cy="Good">Carrot</li>
-      <li data-cy="Good">Eggs</li>
-      <li data-cy="Good">Ice cream</li>
-      <li data-cy="Good">Apple</li>
-      <li data-cy="Good">...</li>
-    </ul>
-  </div>
-);
+export const App = () => {
+  const [sortMetod, setSortMetod] = useState('');
+  const [reversed, setReversed] = useState(false);
```

Comment:
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```

Prefix boolean values with `is` or `has`